.github/workflows/win_no_exception_ci.yml

```diff
@@ -71,7 +71,7 @@ jobs:
         cmake -G "Visual Studio 17 2022" -A ${{ matrix.architecture }} -DONNX_USE_PROTOBUF_SHARED_LIBS=OFF -DONNX_USE_LITE_PROTO=ON -DONNX_WERROR=ON -DCMAKE_BUILD_TYPE=Release -DONNX_USE_MSVC_STATIC_RUNTIME=OFF -DONNX_ML=1 -DONNX_BUILD_TESTS=ON -DONNX_DISABLE_STATIC_REGISTRATION=ON -S . -B .setuptools-cmake-build\
 
         cd .setuptools-cmake-build\
-        msbuild onnx.sln /m /p:Configuration=Release
+        cmake --build . --config Release
```

Comment:
Why do you change from msbuild -> cmake?

Reply:
Because I changed the build script for Windows ONNX.